src/api/resources/collect_record.py

```diff
@@ -228,3 +232,27 @@ def ingest(self, request, project_pk, *args, **kwargs):
             return Response(errors, status=400)
 
         return Response(CollectRecordSerializer(records, many=True).data)
+
+    @action(
+        detail=False,
+        methods=["GET"],
+        permission_classes=[ProjectDataAdminPermission],
+        url_path="ingest_schema/(?P<sample_unit>\w+)",
```

Comment:
Should we append `/csv/` so that we're consistent with the way other endpoints do this, and to allow for a json response without the `/csv/` if we decide to do that?